vpn/src/main/java/com/duckduckgo/mobile/android/vpn/pixels/DeviceShieldPixels.kt

```diff
@@ -378,12 +376,11 @@ interface DeviceShieldPixels {
 @ContributesBinding(AppScope::class)
 @SingleInstanceIn(AppScope::class)
 class RealDeviceShieldPixels @Inject constructor(
-    private val context: Context,
-    private val pixel: Pixel
+    private val pixel: Pixel,
+    vpnSharedPreferencesProvider: VpnSharedPreferencesProvider,
 ) : DeviceShieldPixels {
 
-    private val preferences: SharedPreferences
-        get() = context.getSharedPreferences(DS_PIXELS_PREF_FILE, Context.MODE_MULTI_PROCESS)
+    private val preferences = vpnSharedPreferencesProvider.getSharedPreferences(DS_PIXELS_PREF_FILE, multiprocess = true)
```

Comment:
Just in this case I use the `VpnSharedPreferencesProvider`. I could add it to the other classes that use harmony shared prefs in this PR if necessary. Just being lazy for now

Reply:
I changed my VPN for a change